db/schema.rb

```diff
@@ -20,6 +20,9 @@
     t.integer "score"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.time "game_time"
```

Comment:
My favorite part of reviewing code is seeing new stuff like this. I didn't realize Time was a column type, how cool!

Reply:
The only one I didn't double check in Postico is where I messed up 🥲